http-core/src/main/scala/org/apache/pekko/http/impl/engine/http2/hpack/ByteStringInputStream.scala

```diff
@@ -28,7 +28,7 @@ private[http2] object ByteStringInputStream {
     bs match {
       case cs: ByteString1C =>
         // TODO optimise, ByteString needs to expose InputStream (esp if array backed, nice!)
```

Comment:
Here is a todo, considering optimization here, can it be deleted.
If not relevant,  ignore it.
LGTM

Reply:
refs: https://github.com/apache/incubator-pekko/issues/995

Reply:
Weird that looks like a optimization only done half-way, this whole class looks somewhat useless in its current form, I wonder how it ended up that way.

The reason this does not turn up in profiles here is that the amount of data that is pushed through here is relatively low (not least because HPACK2 is a compression algorithm that limits the amount data necessary).